src/index.html

```diff
@@ -13,6 +13,38 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
```

Comment:
Instead of creating each calendar field manually, consider using JavaScript to generate these fields dynamically. This will make your code more scalable and easier to maintain.